src/scripts/settings/userSettings.js

```diff
@@ -140,7 +140,7 @@ export class UserSettings {
             return this.set('preferFmp4HlsContainer', val.toString(), false);
         }
 
-        return toBoolean(this.get('preferFmp4HlsContainer', false), false);
+        return toBoolean(this.get('preferFmp4HlsContainer', false), true);
```

Comment:
We need to disable fMP4 in device profile on some platforms. _Then we can remove this option (always enable fMP4)._
At least on Tizen 4, I've never been able to get it to work - it always waits for the remuxing job to complete.

Reply:
So disabled for this version of Tizen? Or only enable fMP4 by default only for HLS.js and Safari?

Reply:
We can trim https://github.com/jellyfin/jellyfin-web/blob/1196ae992f3afc27f8cef8e74367f98bb321be31/src/scripts/browserDeviceProfile.js#L71

but agree
> It is unlikely that we will test every version of Tizen and WebOS one by one. I think they should be left to the user's choice.

As an option, `true` -> `!browser.tizen && !browser.web0s` or list the platforms that play fMP4 for sure. In this case it will be disabled by default on some platforms, but the user can still enable it.

Tbh, I don't know why it works that way on my TV. :shrug: Iirc, I tested fMP4 with AVPlay - same bug.

Reply:
I increased the minimum version from 3 to 5 based on your experience.
```diff
-browser.tizenVersion >= 3
+browser.tizenVersion >= 5
```

Do these older Tizen versions support MSE? If so you can enable HLS.js as a replacement.

Reply:
Yeah. Even Tizen 3.0 has been marked as the first version to support fMP4-HLS. Have you also tried the online m3u8 link provided by Apple? If the fMP4 in it cannot be played normally, then this should be a system issue and should be reported to Samsung.

https://developer.samsung.com/smarttv/develop/specifications/general-specifications.html#Streaming-Feature-Support